targets/targets.json

```diff
@@ -7826,15 +7826,16 @@
             "SERIAL",
             "SPI",
             "TRNG",
-            "FLASH"
+            "FLASH",
+            "USTICKER"
         ],
         "public": false,
         "supported_toolchains": ["GCC_ARM", "ARM"],
         "post_binary_hook": {
             "function": "RTL8195ACode.binary_hook",
-            "toolchains": ["ARM_STD", "GCC_ARM", "IAR"]
+            "toolchains": ["ARM_STD", "GCC_ARM"]
```

Comment:
IAR should be also enabled to have "5" in the versions below.

Reply:
The commit "disable IAR" without having a reason is not sufficient - please add more details there.

Reply:
This PR is to bring back basic RTL8195AM support, at least for GCC and ARMCC. IAR will have to be disabled until its HAL library is upgraded to IAR8.

Reply:
Understood, therefore 5 version should be still disabled (not being added in this PR).

Reply:
Will remove that patch, and add details to disable IAR patch. But, users will have to manually add 5 to the release version to enable RTL8195AM support.